### PR TITLE
Merge testing into master

### PR DESCRIPTION
**Pull request by nicofrand**
## Problem

- Migrate to v2 format + Kresus 0.20.0

## Solution

- Merge testing

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: doc/ADMIN.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Limitations
+
+* By default, all users have access to the accounts. The admin needs to manually select the authorised user through YunoHost Administration Panel.
+
+## Roadmap
+
+* to be added:
+  * [ ] URL cannot be changed
+  * [ ] Improve log file and add logrotate
```
Comment (ericgaspar) [inline doc/ADMIN.md:9]: Are those points to be removed?

Comment (nicofrand) [reply doc/ADMIN.md:9]: Hm, I guess I missed it while rebasing. I'll edit in on testing, good catch!

Comment (nicofrand) [reply doc/ADMIN.md]: Fixed! Thanks

File: kresus-0.19.4.tar.bz2.1
[the remark this answers is not in the document]
Comment (nicofrand) [reply kresus-0.19.4.tar.bz2.1:1]: Sh*t. This was an error (downloaded the archives to do a checksum) but thought I removed it already...
Will fix it, thanks.

File: doc/POST_INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Domain: __DOMAIN__
+Path  : __PATH__
+Config: __INSTALL_DIR__/config.ini
+
+Please remember the default behavior of YunoHost when installing a new app is: everyone has access to the app.
```
[the remark this answers is not in the document]
Comment (nicofrand) [reply doc/POST_INSTALL.md:5]: There is no way to make the app available to 1 user only so this is kind of misleading.
Maybe we can reword it to "⚠️ Every YunoHost account will have access to this app".

[the remark this answers is not in the document]
Comment (alexAubin) [reply doc/POST_INSTALL.md:5]: by default, YunoHost will propose : visitors, all_users, admins, and any additional specific group that the admins may have defined, so my point was to clarify that the described behavior is only if the person installing the app picked "all users"

[the remark this answers is not in the document]
Comment (nicofrand) [reply doc/POST_INSTALL.md:5]: OK thanks! I'll amend it

**Review by alexAubin — APPROVED** [review]
